src/packet.proto

```diff
@@ -2,6 +2,23 @@ syntax = "proto3";
 
 package helium;
 
+message gps_info {
+    float horizontal_accuracy = 1; // meters this will take into account h3 precision and distance from asserted location
+    float vertical_accuracy = 2; // meters
+    int32 altitude = 3; // meters
+    uint64 time_accuracy = 4; //nanoseconds
+}
+
+// https://www.eecis.udel.edu/~mills/ntp/html/ntpq.html#system
+message ntp_info {
+    bool synced = 1;
+    uint32 stratum = 2;
```

Comment:
Maybe add a comment here that `stratum` is 1-15 as mentioned by ntp standard. Unsure if both are inclusive.